src/Microsoft.HttpRepl.Tests/OpenApi/EndpointMetadataReaderTests.cs

```diff
@@ -0,0 +1,495 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.HttpRepl.OpenApi;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.OpenApi
+{
+    public class EndpointMetadataReaderTests
+    {
+        [Fact]
+        public void Read_WithInvalidJson_ReturnsNull()
+        {
+            string json = @"{
```

Comment:
nit: can you add a comment explaining what is invalid about this JSON payload?  I'm *assuming* it's invalid because it doesn't have a `"swagger"` property.

Reply:
Updated the test name to Read_WithJObjectFormatNotSupportedByAnyExistingReader_ReturnsNull here: https://github.com/aspnet/HttpRepl/commit/e95a6c7efc23b60ce7efccda7c49fbf286d3e68c

Let me know if it's not clear enough.

Thanks!